packages/datadog-instrumentations/src/mocha.js

```diff
@@ -0,0 +1,91 @@
+const { addHook, channel, AsyncResource } = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const runTestStartCh = channel('ci:mocha:run-test:start')
```

Comment:
This seems a bit redundant having `run` and `start`. It would be more readable to have something like `ci:mocha:test:start`.